<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAlbumDialog</class>
 <widget class="QDialog" name="AddAlbumDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить альбом</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="loadCover">
     <property name="text">
      <string>Загрузить обложку</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelCover">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="text">
      <string>Превью</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTitle">
       <property name="text">
        <string>Название альбома:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTitle"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelArtist">
       <property name="text">
        <string>Исполнитель:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditArtist"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelGenre">
       <property name="text">
        <string>Жанр:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditGenre"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelYear">
       <property name="text">
        <string>Год выпуска:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBoxYear">
       <property name="minimum">
        <number>1900</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
       <property name="value">
        <number>2024</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonBoxOk">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoxCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
